src/app/accessRequests/accessRequests.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const Account = require('./../../infrastructure/account');
+const { getOrganisationUsersForApproval, putUserInOrganisation } = require('../../infrastructure/organisations');
+const flatten = require('lodash/flatten');
+const uniq = require('lodash/uniq');
+const logger = require('./../../infrastructure/logger');
+
+const getUserDetails = async (usersForApproval) => {
+  const allUserId = flatten(usersForApproval.map((user) => user.user_id));
+  const distinctUserIds = uniq(allUserId);
+  return await Account.getUsersById(distinctUserIds);
+};
+
+const get = async (req, res) => {
+  let usersForApproval = await getOrganisationUsersForApproval(req.user.sub, req.id);
+
+  if(usersForApproval) {
+    const userList = await getUserDetails(usersForApproval);
+
+    usersForApproval = usersForApproval.map((user) => {
+      const userFound = userList.find(c => c.claims.sub.toLowerCase() === user.user_id.toLowerCase());
+      const usersName = userFound ? `${userFound.claims.given_name} ${userFound.claims.family_name}` : 'No Name Supplied';
+      const usersEmail = userFound ? userFound.claims.email : '';
+      return Object.assign({usersName, usersEmail}, user);
+    });
+  }
+  res.render('accessRequests/views/requests', {
+    usersForApproval,
+    csrfToken: req.csrfToken(),
+  });
+};
+
+const post = async (req, res) => {
+
+  const userId = req.body.user_id;
+  const orgId = req.body.org_id;
+  const status = req.body.approve_reject.toLowerCase() === 'approve' ? 1 : -1;
+  let role;
+  let reason = req.body.message;
+
+  if(status === -1) {
```

Comment:
This if/else can be reduced to a single `if (status === 1)` by changing the declaration of role to `let role = 0;` which I think would make this more readable